bin/check_multi-line-string-array_contents

```diff
@@ -0,0 +1,38 @@
+#!/bin/sh
+
+# This script checks that multi-line-string-array only contains valid keys.
+
+failed=0
+
+check_keys() {
+    json_file=$1
+    echo "Checking 'multi-line-string-array' contents in $json_file"
+
+    present_keys=$(jq '[ .cases[] | recurse(.cases[]?) | ."multi-line-string-array" | select(. != null)[] ] | unique ' $json_file)
```

Comment:
Since you have specified in canonical-schema.json that `multi-line-string-array` cannot be nested in `labeledTestGroup`, but only directly in `labeledTest` (which is unlike the `optional` parameter that is allowed to occur in both places), the use of `recurse(.cases[]?)` is not necessary and would incorrectly accept changes that break the schema.

It sounds like you were interested in not allowing nesting for the sake of simplicity.

Either way, the schema and the linter should conform.

Also, leading whitespace in `jq` string.

```suggestion
    present_keys=$(jq '[ .cases[] | ."multi-line-string-array" | select(. != null)[] ] | unique' $json_file)
```